EnableGPlayWithPC/Main.cs

```diff
@@ -59,11 +59,28 @@ private void Button_Process_Click(object sender, EventArgs e)
 
                 if (AdbClient.Instance.GetDevices().Count > 1)
                 {
-                    ErrorDialog.ShowError(Properties.Resources.Dialog_TooManyDevices_Inst,
+                    Dialog.ShowError(Properties.Resources.Dialog_TooManyDevices_Inst,
                         Properties.Resources.Dialog_TooManyDevices_Desc, Handle);
                     return;
                 }
 
+                ConsoleOutputReceiver receiver = new ConsoleOutputReceiver();
+
+                AdbClient.Instance.ExecuteRemoteCommand($"getprop ro.build.product", device, receiver);
+                string product = receiver.ToString();
+                product = product.Substring(0, product.Length - 2); // 余計な改行は入れさせない
+
+                Console.WriteLine(product.Length);
+
+                if (!BenesseTabs.Names.Contains(product))
+                { // 出力が名前にあるか確認
+                    TaskDialogResult result =
+                        Dialog.ShowQuestion(Properties.Resources.Dialog_Not_Benesse_Tab_Inst,
+                        string.Format(Properties.Resources.Dialog_Not_Benesse_Tab_Desc, product), Handle);
+
+                    if (TaskDialogResult.Ok != result) return;
```

Comment:
```suggestion
                    if (result != TaskDialogResult.Ok) return;
```
``result``を左辺にしてください。